doc/man3/OSSL_STORE_INFO.pod

```diff
@@ -76,23 +81,25 @@ STORE_INFO_get_type_string() takes a STORE type number and returns a
 short string describing it.
 
 OSSL_STORE_INFO_get0_NAME(), OSSL_STORE_INFO_get0_NAME_description(),
-OSSL_STORE_INFO_get0_PARAMS(), OSSL_STORE_INFO_get0_PKEY(),
-OSSL_STORE_INFO_get0_CERT() and OSSL_STORE_INFO_get0_CRL() all take a
-B<OSSL_STORE_INFO> and return the held object of the appropriate OpenSSL
-type provided that's what's held.
+OSSL_STORE_INFO_get0_PARAMS(), OSSL_STORE_INFO_get0_PUBKEY(),
+OSSL_STORE_INFO_get0_PKEY(), OSSL_STORE_INFO_get0_CERT(),
+OSSL_STORE_INFO_get0_CRL()
+all take a B<OSSL_STORE_INFO> and return the held object of the appropriate
+OpenSSL type provided that's what's held.
```

Comment:
'provided that's what's held' should maybe be replaced with something like
'or NULL if the object does not exist'..

Reply:
In that case, 'or NULL if the B<OSSL_STORE_INFO> holds an object of a different type'

Reply:
I rephrased it a bit more